src/main/java/roomescape/domain/Reservation.java

```diff
@@ -16,6 +16,13 @@ public Reservation(Long id, String name, LocalDate date, ReservationTime time) {
         this.time = time;
     }
 
+    public Reservation(String name, LocalDate date, ReservationTime time) {
+        this.id = null;
+        this.name = name;
+        this.date = date;
+        this.time = time;
+    }
```

Comment:
부생성자라면 this()를 통해 주생성자를 호출해줍시다~!
```suggestion
    public Reservation(String name, LocalDate date, ReservationTime time) {
        this(null, ...);
    }
```